ops/terraform/modules/resources/dns_pairs/main.tf

```diff
@@ -0,0 +1,37 @@
+# Create weighted A-record pairs with 0-100 for weights. 
+#
+resource "aws_route53_record" "record_a" {
+  for_each  = var.weights
+  zone_id   = var.zone_id
+  name      = each.key
+  type      = "A"
```

Comment:
Do these need to be CNAME records since the alias is an FQDN instead of an IP address?

Reply:
Good question. Can you ping me offline about this. Since the target is an Alias, I think A records work. At least they do in my testing. It saves a step in DNS resolution.

Reply:
My mistake, R53 will resolve the alias to an IP before responding.